src/Microsoft.AspNetCore.Mvc.RazorPages/ApplicationModels/PageRouteModel.cs

```diff
@@ -21,9 +21,21 @@ public class PageRouteModel
         /// <param name="relativePath">The application relative path of the page.</param>
         /// <param name="viewEnginePath">The path relative to the base path for page discovery.</param>
         public PageRouteModel(string relativePath, string viewEnginePath)
+            : this(relativePath, viewEnginePath, areaName: null)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="PageRouteModel"/>.
+        /// </summary>
+        /// <param name="relativePath">The application relative path of the page.</param>
+        /// <param name="viewEnginePath">The path relative to the base path for page discovery.</param>
```

Comment:
This is also relative to the area root right?

Reply:
Yup. I'll expand on what this is and say that this is the equivalent of pageName

Reply:
Added docs as part of the property description.